plugin/pkg/scheduler/algorithm/priorities/util/non_zero.go

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
Really?
Is pkg/api or pkg/api/resource somehow transitively importing anything from scheduler?
Can you paste this cycle? [It definitely shouldn't be the case]